src/resources/ble_esp32.cc

```diff
@@ -294,13 +295,15 @@ class BleCallbackResource : public BleResource {
 
   void* token() const {
     if (token_ == kInvalidToken) FATAL("Ble Resource token wasn't set");
+    if (token_ == kDeletedToken) FATAL("Ble Resource token taken for deleted object");
```

Comment:
I think I'd prefer "BLE resource token ..." or "BleResource token ..." here and in other places.

Reply:
done.